apps/grader-host/src/index.ts

```diff
@@ -214,19 +211,14 @@ async function handleJob(job) {
   }
 }
 
-/**
- * @typedef {Object} Context
- * @property {Docker} docker
- * @property {S3} s3
- * @property {import('./lib/jobLogger.js').WinstonBufferedLogger} logger
- * @property {import('./lib/receiveFromQueue.js').GradingJobMessage} job
- */
-
-/**
- * @param {Context} context
- * @param {Date} receivedTime
- */
-async function reportReceived(context, receivedTime) {
+interface Context {
+  docker: Docker;
+  s3: S3;
+  logger: import('./lib/jobLogger.js').WinstonBufferedLogger;
```

Comment:
Do you want to import this?

Reply:
Yep - this was a bad merge conflict resolution.

Reply:
Fixed in d7e7403.